pkg/asset/installconfig/azure/validation.go

```diff
@@ -230,6 +246,11 @@ func ValidateInstanceType(client API, fieldPath *field.Path, region, instanceTyp
 	allErrs = append(allErrs, validateMininumRequirements(fieldPath.Child("type"), req, instanceType, capabilities)...)
 	allErrs = append(allErrs, validateVMArchitecture(fieldPath.Child("type"), instanceType, architecture, capabilities)...)
 
+	family, _ := client.GetVirtualMachineFamily(context.TODO(), instanceType, region)
+	if family != "" {
+		allErrs = append(allErrs, validateFamily(fieldPath.Child("type"), instanceType, family)...)
+	}
+
```

Comment:
Couldn't it be useful here to at least log any errors from `GetVirtualMachineFamily` for debugging?
```suggestion
	family, err := client.GetVirtualMachineFamily(context.TODO(), instanceType, region)
	if family != "" {
		allErrs = append(allErrs, validateFamily(fieldPath.Child("type"), instanceType, family)...)
	}
	if err != nil {
	    logrus.Debugf("failed to retrieve instance family: %v", err)
	}

```

Reply:
Oh my bad. Disregard then.

Reply:
My previous suggestion was to log a warning but not throw an error: https://github.com/openshift/installer/pull/6203#discussion_r950123108

But I think existing code is fine.